### PR TITLE
Initial commit of branding

### PR DESCRIPTION
**Pull request by jeffkl**
Just a few logos and a README  explaining everything.  I'll update the forwarding link once this is merged.

Please view the README to see the icons in action.

### CODE REVIEW

File: branding/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+This folder contains images used for branding MSBuild in various places.  If you add a new image, please indicate where it is used.
+
+## MSBuild-Icon.png ![MSBuild-NuGet-Icon.png](MSBuild-Icon.png?raw=true)
+This image is the original graphic used for creating the MSBuild icon.  It was placed here from it's original location but is not currently used by anything.
+
+## MSBuild-Logo.png ![MSBuild-NuGet-Icon.png](MSBuild-Logo.png?raw=true)
+This image is an original graphic created by Immo Landwerth.  It can be used for images where the size is required to be larger than the MSBuild-Icon.png.
+
+## MSBuild-NuGet-Icon.png ![MSBuild-NuGet-Icon.png](MSBuild-NuGet-Icon.png?raw=true)
+This image is used by the NuGet packages as the ``<iconUrl />``.  It's raw location in this repo is used by the forwarding link [https://go.microsoft.com/fwlink/?linkid=825694](https://go.microsoft.com/fwlink/?linkid=825694).  If it is moved, please update the link.
```
Comment (rainersigwald) [inline branding/README.md]: typo: it's -> its

File: branding/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+If the below image loads, then the forwarding link is working.
```
Comment (rainersigwald) [inline branding/README.md]: 🎉

**Comment by rainersigwald** [comment]
LGTM. We also need to get the `.ico` icon and embed it in the binary we produce from GitHub but that's not too urgent.

File: branding/README.md (hunk 1 above)
Comment (jeffkl) [inline branding/README.md]: Fixed

**Pull request merged**